tools/crm_resource_runtime.c

```diff
@@ -1508,6 +1512,34 @@ cli_resource_execute(const char *rsc_id, const char *rsc_action, GHashTable *ove
 
     params = generate_resource_params(rsc, data_set);
 
+    /* add meta_timeout env needed by some resource agents */
```

Comment:
crm_resource has a --timeout option; if the user sets it, I think we should use it, and skip the configuration check